src/components/Base/Base.tsx

```diff
@@ -74,23 +75,25 @@ const getFieldErrors = (
   if (error.category === 'invalid_attribute_value') {
     return [
       {
-        key: keyPrefix + error.errorKey,
+        key: snakeCaseToCamelCase(keyPrefix + error.errorKey),
         message: error.message ?? '',
       },
     ]
   }
   if (error.category === 'nested_errors' && error.errors !== undefined) {
     //TODO: clean this up once Metadata type is fixed in openapi spec
-    const keySuffix =
+    let keySuffix = ''
```

Comment:
Broke this out from a complicated nested ternary to a if/else function as its a bit more clear to read. I had to add another logical branch so figured this made sense.